hycom/RELO/src_2.2.98ZA-07Tsig0-i-sm-sse_relo_mpi/thermf.F

```diff
@@ -14,6 +14,7 @@
       subroutine thermf_oi(m,n)
       use mod_xc         ! HYCOM communication interface
       use mod_cb_arrays  ! HYCOM saved arrays
+      use mod_NERSCnml, only : sss_underice, nersc_init
```

Comment:
You dont need to include nersc_init as it is not called from therm.F

Reply:
Thanks for this comment. It can be deleted.